### DISCUSSION TITLE
View source and time in a single row in article list

### DISCUSSION THREAD

**Discussion by sozifa** [jocmp/capyreader#326] [Feature Requests]
It would make a more compact layout to show source and time in just one row instead of two
Cheers

**Comment by jocmp** [comment]
Tracking via https://github.com/jocmp/capyreader/issues/330. No timeline yet.

**Reply by jocmp** [reply]
This is available in [2024.09.1042-dev](https://github.com/jocmp/capyreader/releases/tag/2024.09.1042-dev). Example below.

![Screenshot 2024-09-08 at 2 53 53 PM](https://github.com/user-attachments/assets/ccc41cc3-6f2c-49a7-bfba-de13edd88733)

**Reply by sozifa** [reply]
Many thanks - just one little bug -, in an article without a thumbnail it now uses the full row - looks a bit weird 
![Screenshot_20240908-232108](https://github.com/user-attachments/assets/158a3181-0236-4ddf-ab01-25e787fe3ec3)

**Reply by jocmp** [reply]
thanks @sozifa. I put up a fix in [2024.09.1043-dev](https://github.com/jocmp/capyreader/releases/tag/2024.09.1043-dev).

Like the picture below, this moves the time to an overline position so that it will always show up in the same place regardless of the image. This is a similar approach to a desktop app I used called [Reeder](https://reederapp.com/).

![Screenshot_20240908_170659](https://github.com/user-attachments/assets/40bb40bc-6eff-4e31-a120-f5bb24c1d3ed)

**Reply by sozifa** [reply]
Works fine for me - I like the new layout even better than before
